channels/vn.m3u

```diff
@@ -1,130 +1,1075 @@
+#EXTINF:-1 tvg-id="ThanhHoa.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Thanh Hóa (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/thanhhoa_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ThanhHoa.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Thanh Hóa (720p)
+http://123.31.36.68/live.m3u8?c=vstv208&q=auto&time=1&requestTime=1&pkg=pkg11
+#EXTINF:-1 tvg-id="TienGiang.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Tiền Giang (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sda/tiengiang_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="TienGiang.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Tiền Giang (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/tiengiang_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="TienGiang.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Tiền Giang (1080p)
+http://123.25.238.45:1935/live/thtg/playlist.m3u8
+#EXTINF:-1 tvg-id="TienGiang.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Tiền Giang (720p)
 http://thtg.vn:8001/thtg.m3u8
-#EXTINF:-1 tvg-id="UniquelyThai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Uniquely Thai (720p)
-https://livefta.malimarcdn.com/ftaedge00/uniquely.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="VajtswvTxojlus.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus (720p)
-https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="VanphenhTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vanphenh TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/vanphenhtv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="VatiLaoTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vati Lao TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/vatilaotv.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="VietMyTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://s30.postimg.org/s0aco3hzl/VIET_MY.png" group-title="",Việt Mỹ TV (480p) [Not 24/7]
-http://68.235.37.11:1935/vietmagazine/vietmagazine/playlist.m3u8
-#EXTINF:-1 tvg-id="VinhLong1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 1 [Offline]
-http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL1-HD-1080p/playlist.m3u8
-#EXTINF:-1 tvg-id="VinhLong2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 2 [Offline]
-http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL2-HD-1080p/playlist.m3u8
-#EXTINF:-1 tvg-id="VITV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/vitv.png" group-title="",VITV (180p)
-http://210.86.230.202:8134/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8
```

Comment:
Working link

Reply:
#EXTINF:-1 tvg-id="VinhLong2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 2 [Offline]
http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL2-HD-1080p/playlist.m3u8
Not working
![image](https://user-images.githubusercontent.com/74099190/140298738-74280faa-50eb-4cc5-a8de-e1c386a33322.png)

Reply:
#EXTINF:-1 tvg-id="VITV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/vitv.png" group-title="",VITV (180p)
http://210.86.230.202:8134/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8

Update later

Reply:
As I said before, all changes must be included in the pull request in order for us to approve it.